<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">

    <xacro:property name="joint_limits_path" value="$(find iiwa_description)/config/joint_setting.yaml"/>
    <xacro:property name="joint_limits" value="${xacro.load_yaml(joint_limits_path)}"/>
    
    
    <xacro:property name="robot_name" value="lbr_iiwa7"/>
    <xacro:property name="PI" value="3.1415926535897931" />
    <xacro:property name="joint_damping" value="10.0" />
    <xacro:property name="joint_friction" value="0.1" />
    
</robot>